docker/mcp_server.py

```
@@ -0,0 +1,224 @@
+        serper_url = "https://google.serper.dev/search"
+        headers = {
+            'X-API-KEY': api_key,
+            'Content-Type': 'application/json'
+        }
+        payload = {
+            'q': query,
+            'gl': 'us',
+            'hl': 'en'
+        }
+        
+        response = requests.post(serper_url, headers=headers, json=payload)
```

Pylint (3.3.7)

[warning] Missing timeout argument for method 'requests.post' can cause your program to hang indefinitely

(W3101)